PoGo.NecroBot.Logic/Tasks/UseNearbyPokestopsTask.cs

```diff
@@ -450,73 +461,38 @@ private static bool SearchThresholdExceeds(ISession session)
-        //to only find stops within 40 meters for GPX pathing, as we are not going to the pokestops,
-        //so do not make it more than 40 because it will never get close to those stops.
-        //For non GPX pathing, it returns all pokestops in range.
-        private static async Task<Tuple<List<FortData>, List<FortData>>> GetPokeStops(ISession session)
-        {
-            List<FortData> mapObjects = await UpdateFortsData(session);
-            session.AddForts(mapObjects);
-
-            if (!session.LogicSettings.UseGpxPathing)
-            {
-                if (mapObjects.Count <= 0)
-                {
-                    // only send this for non GPX because otherwise we generate false positives
-                    session.EventDispatcher.Send(new WarnEvent
-                    {
-                        Message = session.Translation.GetTranslation(TranslationString.FarmPokestopsNoUsableFound)
-                    });
-                }
-
-                var pokeStops = mapObjects.Where(p => p.Type == FortType.Checkpoint).ToList();
-                session.AddVisibleForts(pokeStops);
-                session.EventDispatcher.Send(new PokeStopListEvent { Forts = mapObjects });
-
-                var gyms = mapObjects.Where(p => p.Type == FortType.Gym).ToList();
-                //   session.EventDispatcher.Send(new PokeStopListEvent { Forts = mapObjects });
-                return Tuple.Create(pokeStops, gyms);
-            }
-
-            if (mapObjects.Count > 0)
-            {
-                // only send when there are stops for GPX because otherwise we send empty arrays often
-                session.EventDispatcher.Send(new PokeStopListEvent { Forts = mapObjects });
-            }
-            // Wasn't sure how to make this pretty. Edit as needed.
-            return Tuple.Create(
-                mapObjects.Where(
-                    i => i.Type == FortType.Checkpoint &&
-                        ( // Make sure PokeStop is within 40 meters or else it is pointless to hit it
-                            LocationUtils.CalculateDistanceInMeters(
-                                session.Client.CurrentLatitude, session.Client.CurrentLongitude,
-                                i.Latitude, i.Longitude) < 40) ||
-                        session.LogicSettings.MaxTravelDistanceInMeters == 0
-                ).ToList(),
-                mapObjects.Where(p => p.Type == FortType.Gym && LocationUtils.CalculateDistanceInMeters(
-                                session.Client.CurrentLatitude, session.Client.CurrentLongitude,
-                                p.Latitude, p.Longitude) < 40).ToList()
-                );
-        }
-
         public static async Task<List<FortData>> UpdateFortsData(ISession session)
         {
             var mapObjects = await session.Client.Map.GetMapObjects();
-            session.AddForts(mapObjects.Item1.MapCells.SelectMany(p => p.Forts).ToList());
```

Comment:
can I ask you a question? if bot walk out of range for snipe, then what poketop it can spin when walking of walk back, you basically just focus on 1 bug  only. that is bug about people complain bot not respect max distance, you even don't know why bot go out of that distance. If you don't trust me, go a head merge it then see people complaint about bus has been fixed before.